packages/app/src/components/inspector/TextAlignmentRow.tsx

```diff
@@ -0,0 +1,109 @@
+import {
+  StretchHorizontallyIcon,
+  StretchVerticallyIcon,
+  SquareIcon,
+  TextAlignCenterIcon,
+  TextAlignJustifyIcon,
+  TextAlignLeftIcon,
+  TextAlignRightIcon,
+  PinBottomIcon,
+  PinTopIcon,
+  AlignCenterVerticallyIcon,
+} from '@radix-ui/react-icons';
+import { Label, RadioGroup, LabeledElementView } from 'noya-designsystem';
+import { Spacer } from 'noya-designsystem';
+import { memo } from 'react';
+import * as InspectorPrimitives from './InspectorPrimitives';
+
+interface TextAligmentRowProps {
+  fontAlignment: number;
```

Comment:
For naming, let's call these:

`textLayout`
`textHorizontalAlignment`
`textVerticalAlignment`